src/schema/rules/tabular_data/nirs.yaml

```diff
@@ -0,0 +1,64 @@
+---
+nirsChannels:
+  selectors:
+    - datatype == "nirs"
+    - suffix == "channels"
+    - extension == ".tsv"
+  initial_columns:
+    - name__channels
+    - type__nirs_channels
+    - source__channels
+    - detector__channels
+    - wavelength_nominal
+    - units__nirs
```

Comment:
There was nothing in the spec text that said that the order of these columns was intended to be enforced. Is the intention that these be enforced? This section could be shortened or dropped, if the order of columns isn't important.

Side question: I assume the `name` column is intended to have unique values. Is that correct?

Reply:
Thanks @effigies